tests/unit/test_cli.py

```diff
@@ -179,6 +179,18 @@ def testReadsSearchArguments(self):
         self.assertEqual(args.baseUrl, "BASEURL")
         self.assertEquals(args.runner, cli.SearchReadsRunner)
 
+    def testBioSamplesSearchArguments(self):
```

Comment:
missing a lot of other cli tests here, right?  bio sample get, repo manager add and remove